docs/tutorials/tutorial-csv.md

```diff
@@ -0,0 +1,51 @@
+This tutorials shows a complete example of using a local COOL package to load the sample sogamo csv dataset and executes a query.
+
+After building the COOL system, under the root directory, execute the following command will ingest the sogamo dataset to COOL system.
+```bash
+java -cp ./target/cool-0.1-SNAPSHOT.jar com.nus.cool.loader.LocalLoader sogamo sogamo/table.yaml sogamo/dim_test.csv sogamo/test.csv ./test 65536
+```
+The command indicates that we use the LocalLoader to load the sogamo dataset. The table.yaml describing the table schema can be found at `sogamo/table.yaml`. The dimension file can be found at `sogamo/dim_test.csv`. The output directory to store the converted dataset is `test` and chunk size for COOL [cublet](../data-formats.md) is 64MB. 
```

Comment:
Please explicitly explain the parameters. 
For example,
`./target/cool-0.1-SNAPSHOT.jar `: xxx;
`com.nus.cool.loader.LocalLoader`: xxxx;
...